scripts/commit-release-assets.sh

```
@@ -25,9 +25,10 @@ git config user.email 'auto@akadenia.com'
 git checkout -b $BRANCH_NAME
 
 # Commit changes
-git add package.json package-lock.json CHANGELOG.md
+git add package.json pnpm-lock.yaml CHANGELOG.md
 git commit -m "chore(release): add release assets from $VERSION"
 
-# Push branch to the remote
+# Push branch to the remote using GH_TOKEN for authentication
+git remote set-url origin "https://${GH_TOKEN}@github.com/akadenia/AkadeniaLogger.git"
 git push origin $BRANCH_NAME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)